radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/transport/PeerChannel.java

```diff
@@ -219,10 +219,12 @@ private void finalizeHandshake(AuthHandshakeResult handshakeResult) {
 	}
 
 	private void handleMessage(ByteBuf buf) throws IOException {
+		final var receiveTime = System.currentTimeMillis();
+
```

Comment:
Please, use `Time` class as the source of timestamp. You may refer to `Message` class for details.